doc/doxygen/Makefile

```
@@ -9,10 +11,35 @@ export STRIP_FROM_INC_PATH_LIST=$(shell \
 # It can also be installed in ubuntu with the `node-less` package
 LESSC ?= lessc
 
-DOCUMENTATION_FORMAT ?= html
+# Extract the documentation type that is to be generated and export it for
+# the doxygen Makefile
+ifneq ($(filter doc,$(MAKECMDGOALS)),)
```

```suggestion
ifneq (,$(filter doc,$(MAKECMDGOALS)))
```

while there's no semantic difference, I'd prefer this order. I think that is also what is currently mostly used in our Makefiles.

(same below)